HotelManagement/src/HotelServiceApplication.java

```diff
@@ -1,25 +1,51 @@
-import core.Hotel;
-import core.Manager;
-import core.Room;
+import core.*;
+import core.Commodities.Bed;
+import core.Commodities.Shower;
+import core.Commodities.Toilet;
 
-import java.util.ArrayList;
+import java.time.LocalDate;
 
```

Comment:
In the application class try separating code blocks with comments for each feature you are testing out.

Reply:
Done